dirsync/syncer.py

```diff
@@ -158,9 +158,9 @@ def _compare(self, dir1, dir2):
 
                 if add_path:
                     left.add(path)
-                    anc_dirs = re_path[:-1].split('/')
+                    anc_dirs = re_path.split('/')
                     anc_dirs_path = ''
-                    for ad in anc_dirs[1:]:
```

Comment:
I guess we have to strip the root directory as we don't want it to appear in the `left` list. Same, this dates from when the module was completely rewritten back in 2014 and it looks like I did not write enough comments! Does stripping the root create issues? If yes, I would be happy to see a failing test case added to the test suite.